data/releases/schedule.yaml

```
@@ -4,6 +4,15 @@
 # schedule-builder -uc data/releases/schedule.yaml -e data/releases/eol.yaml
 ---
 schedules:
+- endOfLifeDate: "2026-06-30"
+  maintenanceModeStartDate: "2026-04-30"
```

It looks like, based on the [support policy (here)](https://kubernetes.io/releases/patch-releases/#support-period), the end and maintenance dates are typically set to the 28th of the month - might be worth aligning with that.

```suggestion
- endOfLifeDate: "2026-06-28"
  maintenanceModeStartDate: "2026-04-28"
```

---

Ah! right. Updated.